packages/lesswrong/components/analytics/MyAnalyticsPage.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import { useCurrentUser } from '../common/withUser';
+import { userGetAnalyticsUrl } from '../../lib/collections/users/helpers';
+
+const MyAnalyticsPage = () => {
+  const {PermanentRedirect, SingleColumnSection, LoginForm} = Components;
+  const currentUser = useCurrentUser();
+
+  if (!currentUser) {
+    return (
+      <SingleColumnSection>
+        <LoginForm />
+      </SingleColumnSection>
+    );
+  }
+
+  return <PermanentRedirect url={userGetAnalyticsUrl(currentUser)}/>
```

Comment:
It will only get this far if logged in so it's fine for this to be permanent

Reply:
But the browser might be used for multiple users. 🤔

Reply:
Good point, I'll make it temporary